planning/behavior_velocity_planner/autoware_behavior_velocity_planner/src/node.cpp

```diff
@@ -305,9 +281,20 @@ bool BehaviorVelocityPlannerNode::processData(rclcpp::Clock clock)
     is_ready = false;
   }
 
+  const auto map_data = sub_lanelet_map_.takeData();
+  if (map_data) {
+    planner_data_.route_handler_ = std::make_shared<route_handler::RouteHandler>(*map_data);
+  }
```

Comment:
NOTE: this heavy process is called only once because PollingSubscriber policy is NEWEST